routes/camps_routes.js

```diff
@@ -18,12 +18,47 @@ module.exports = function (app, passport) {
             name: 'camps:breadcrumbs.home',
             url: '/' + req.params.lng + '/camps'
         });
-        // TODO - add api call to test if user is part of camp 
-        // if user is member of camp : res.redirect => camp-profile-page
-        res.render('pages/camps/index_user', {
-            user: req.user,
-            breadcrumbs: req.breadcrumbs()
-        });
+        if (req.user.myCamps((camps) => {
```

Comment:
then again, there is no need.
just had in mind that it could be Promised function and not thru call back.
:)